### PR TITLE
Update maps control docs

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## What is the current behavior?
Documentation for Maps is a bit outdated

## What is the new behavior?
Updated documentation and added relevant information

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information
This update was inspired after reading a question post on SO and the issues the OP had. 

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/controls/map-control-support.md [hunk 1]
```diff
@@ -56,65 +58,74 @@ See a more complete sample here:
...
+    2.  Add the API key to `AssemblyInfo.cs`.
+    
+    ```csharp
+	[assembly: MetaData("com.google.android.maps.v2.API_KEY", Value = "YOUR_API_KEY")]
+    ```
+    Replace the text YOUR_API_KEY with the key generated in previous step.
+    
+    Note: Since this key might vary depending on the platform and environment we suggest using a constant class where the key could be retrieved from.
+	
+    3.  Add the relevant permissions. For example, if you wish to access the user location
```
Comment (MatFillion) [inline doc/articles/controls/map-control-support.md]: Should mention where to add them. Your example is in c# so I'm guessing AssemblyInfo.cs like the APIKey of step  2?
Most people are used to add them in xml in the manifest directly, but I feel it's cleaner in C# and easier to add comments/notes without VisualStudio messing it all up.

File: doc/articles/controls/map-control-support.md [hunk 2]
```diff
@@ -56,65 +58,74 @@ See a more complete sample here:
...
+    
+	 ```csharp
+	[assembly: UsesPermission(Android.Manifest.Permission.AccessFineLocation)]
+	[assembly: UsesPermission("com.myapp.permission.MAPS_RECEIVE")]
+	[assembly: Permission(Name = "com.myapp.permission.MAPS_RECEIVE", ProtectionLevel = Android.Content.PM.Protection.Signature)]
+	```
+        
+**Important:** For both iOS and Android make sure to install the [Uno.UI.Maps](https://www.nuget.org/packages/Uno.UI.Maps/) NuGet package.
```
Comment (MatFillion) [inline doc/articles/controls/map-control-support.md]: That would be better on top of the file? :D Most people don't read til the end!

File: doc/articles/controls/map-control-support.md (hunk 1 above)
Comment (ajpinedam) [reply doc/articles/controls/map-control-support.md]: Updated

File: doc/articles/controls/map-control-support.md (hunk 2 above)
Comment (ajpinedam) [reply doc/articles/controls/map-control-support.md]: Updated adding a session for this.

**Pull request merged**